R/host.R

```diff
@@ -141,3 +149,19 @@ schedule_handle_message_from_server <- function() {
 drop_nulls <- function(x) {
   x[!vapply(x, is.null, FUN.VALUE = logical(1))]
 }
+
+# Enough information for the user to be able to identify which
+# session is which when using `list_r_sessions()` (#18)
+describe_session <- function() {
+  paste0(the$i, ": ", basename(getwd()), " (", infer_ide(), ")")
```

Comment:
```suggestion
 sprintf("%d: %s (%s)", the$i, basename(getwd()), infer_ide())
```
I think this reads more clearly?